src/components/ExpandCollapse/ExpandCollapse.scss

```diff
@@ -32,12 +32,6 @@
     color: var(--icon-color);
   }
 }
-
-.expand-collapse__header--open {
```

Comment:
just making sure it's ok to remove from the non-module file

Reply:
It will only be a problem if there are some mf users who haven't updated their vibe version to 1.86.0 or higher and are using the Expand collapse / Accordion components. However, this issue is already happening for those users who didn't update, because of a previous fix that we implemented in the same area, and it's already part of the latest version of vibe package.

If a mf hasn't updated their version, the bottom border of their accordion/expand collapse components may be hidden (which isn't a frequently used component anyway) until they update their version.

Reply:
It's worth mentioning that we regularly deal with CSS bugs due to conflicts between the CSS on the MFS and the monolith. We can't avoid these conflicts until we complete the CSS module migration because in not completely migrated components, different versions are using the same class names.